docs/sources/mimir/configure/about-versioning.md

```diff
@@ -173,7 +173,7 @@ The following features are currently experimental:
   - Maximum response size for active series queries (`-querier.active-series-results-max-size-bytes`)
   - Enable PromQL experimental functions (`-querier.promql-experimental-functions-enabled`)
   - Allow streaming of `/active_series` responses to the frontend (`-querier.response-streaming-enabled`)
-  - Mimir query engine (`-querier.query-engine=mimir` and `-querier.enable-query-engine-fallback`, and all flags beginning with `-querier.mimir-query-engine`)
+  - [Mimir query engine]({{< relref "../references/architecture/mimir-query-engine" >}}) (`-querier.query-engine=mimir` and `-querier.enable-query-engine-fallback`, and all flags beginning with `-querier.mimir-query-engine`)
```

Comment:
We should use a full URL with version substitution syntax rather than a relref here. Relrefs are deprecated. See https://grafana.com/docs/writers-toolkit/write/links/#link-to-grafanacom-pages.